wbia/algo/preproc/preproc_image.py

```diff
@@ -116,7 +116,7 @@ def islocal(gpath):
                         ), '200 code not received on download'
                     except Exception:
                         scheme = urlsplit(uri_, allow_fragments=False).scheme
-                        uri_ = uri_.strip('%s://' % (scheme,))
+                        uri_ = uri_[len('%s://' % (scheme,)) :]
                         uri_path = urlquote(uri_.encode('utf8'))
```

Comment:
@bluemellophone `uri_` was already "urlquoted" on line 109.  Do we really want to urlquote it again?

The exception handling code was added in commit 1e9391c6ff "Add support for poorly encoded images".  Do you remember what you were trying to fix?